android/app/build.gradle

```diff
@@ -73,13 +73,14 @@ import com.android.build.OutputFile
  */
 
 project.ext.react = [
-    entryFile: "index.js",
-    bundleConfig: System.getenv('BUNDLE_CONFIG')
+        entryFile: "index.js",
```

Comment:
Looks like to much indent but not important :man_shrugging:

Reply:
I copied this from the template. I think it is best to stick to the indentation of the templates for these files. Else it is possible to get merge conflicts when upgrading rn.